src/comparator/package.json

```diff
@@ -16,11 +16,9 @@
   "dependencies": {
     "@build-tracker/formatting": "^1.0.0-beta.3",
     "@build-tracker/build": "^1.0.0-beta.3",
+    "@build-tracker/types": "^1.0.0-beta.3",
     "markdown-table": "^1.1.2"
   },
-  "devDependencies": {
-    "@build-tracker/types": "^1.0.0-beta.3"
-  },
```

Comment:
Since the declaration files depend on this, we need to make sure this dependency is installed (outside of dev).

Reply:
We might also need to do this for all `@types` 🤔

Reply:
I just took a glance at jest's config and it looks like they're using declarations (https://github.com/facebook/jest/blob/master/tsconfig.json). Maybe we can copy whatever they're doing?

Reply:
I did some digging. It looks like they're using [TS project references](https://www.typescriptlang.org/docs/handbook/project-references.html). I experimented with doing the same in https://github.com/paularmstrong/build-tracker/pull/117. It seems to work but we'd need to roll it out to all packages.

Reply:
Would that be possible at the root level config, since all of the workspaces inherit it?